src/CheckRightsService.Broker/CheckRightsService.Broker.csproj

```diff
@@ -10,4 +10,8 @@
     <ProjectReference Include="..\CheckRightsService.Data\CheckRightsService.Data.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="LT.DigitalOffice.Kernel" Version="1.1.9" />
```

Comment:
Probably, you should not write it here, as I know, we have targets in project, there are all packages. I may be wrong

Reply:
fixed